pallets/compliance-manager/src/lib.rs

```diff
@@ -118,11 +122,32 @@ pub trait Trait:
 }
 
 use polymesh_primitives::condition::ConditionOld;
+/// Implicit requirement status.
+#[cfg_attr(feature = "std", derive(Serialize, Deserialize))]
+#[derive(Encode, Decode, Clone, Copy, PartialEq, Eq, Debug)]
+pub enum ImplicitRequirementStatus {
+    /// It means all the implicit requirements will be checked
+    /// at the time of compliance check for a transfer of a given asset.
+    /// Ex - ConditionType::HasValidProofOfInvestor is a implicit requirement
+    /// for sender and receiver both.
+    ///
+    /// By default every asset has the implicit requirement active.
+    Active,
+    /// No implicit requirements will be checked during the asset transfer.
+    Inactive,
+}
```

Comment:
A boolean would've sufficed here :)

Reply:
True, I'm personally a fan of not using `bool` though for the added readability and type safety a dedicated 2-variant `enum` adds.

Reply:
And another reason is that by default I need to set the value true (i.e Active) that is not possible for a map if I am not wrong or go with negating functionality but it seems wonky at the time code read so I choose to have enum variant which is straight forward.

Reply:
An extra Debug instance in the WASM runtime?